pinot-plugins/pinot-input-format/pinot-protobuf/pom.xml

```diff
@@ -50,6 +50,11 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Any specific reason why we need to override this version? This can potentially cause version conflict.

Reply:
No. Only pinot-query-planner is using the org.codehaus.janino dependency. I's also using 3.1.6 version. There's no common version so I specified it here as well

Reply:
We have pinned its version to `3.1.12` in #12714. Maybe you didn't pull the latest code when merging this